bagit.py

```diff
@@ -1499,6 +1527,17 @@ def _make_parser():
             " without performing checksum validation to detect corruption."
         ),
     )
+    parser.add_argument(
+        "--wrap",
```

Comment:
I like the idea but I was wondering whether “wrap” might be a little too generic a name for the option. What do you think about something like `--wrap-tags` or `--max-tag-line-length`?

Reply:
Yeah, I struggled with naming the option! The current result was the outcome of trying to satisfy two goals: (1) emulating the existing `bagit.py` style of using mostly single words (for those options other than the metadata fields), and (2) accounting for the fact that the argument parser displays the associated variable name as part of the help text. The latter means that one ends up having to think about the argument name and the variable jointly. The current combination leads to the following output for `bagit.py -h`:

```
--wrap LINE_LENGTH    Limit line lengths in the tag file (bag-info.txt) by
                      wrapping long values and indenting subsequent lines
                      with a space character. (Default: don't.)
```

I had hoped this was clear enough, but I have to agree that by itself, the current combination leaves ambiguous _what_ is being wrapped. I'm fine with changing it. I lean towards `--wrap-tags`.

Reply:
+1 on `--wrap-tags`. I'd prefer clarity to single words and with that we don't to touch either `dest` or add a `metavar` option.

Reply:
(Just mentioning it in case it wasn't obvious: all these discussed changes are in the update to the PR made a few days ago.)